src/pages/ShopDetail/RestaurantReview/RestaurantReview.js

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import './RestaurantReview.scss';
+import ReveiwGradeButton from '../ReveiwWritingPage/ReveiwGradeButton/ReveiwGradeButton';
```

Comment:
- import 속성 순서 신경써주세요!
- `React → Library(Package) → Component → 변수 / 이미지 → css 파일(scss 파일)`
- css 는 가장 마지막 입니다!
```suggestion
import React from 'react';
import ReveiwGradeButton from '../ReveiwWritingPage/ReveiwGradeButton/ReveiwGradeButton';
import './RestaurantReview.scss';
```